pkg/app/web/src/components/app-sync-status.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC } from "react";
+import { Box, makeStyles, Typography } from "@material-ui/core";
+import { SyncStatusIcon } from "./sync-status-icon";
+import { APPLICATION_SYNC_STATUS_TEXT } from "../constants/application-sync-status-text";
+import {
+  ApplicationSyncState,
+  ApplicationSyncStatus,
+} from "../modules/applications";
+
+const useStyles = makeStyles((theme) => ({
+  statusText: {
+    marginLeft: theme.spacing(0.5),
+  },
+}));
+
+interface Props {
+  syncState?: ApplicationSyncState;
+  deploying: boolean;
+  className?: string;
+  size?: "medium" | "large";
+}
+
+const NOT_AVAILABLE_TEXT = "N/A";
```

Comment:
nit, similar to [pkg/app/web/src/components/application-list-item.tsx#L52](https://github.com/pipe-cd/pipe/blob/master/pkg/app/web/src/components/application-list-item.tsx#L52) should we move it to `constants/text.ts` :eyes:

Reply:
You're right, thanks😄